reaper.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/sirupsen/logrus"
+)
+
+func forkExec() {
+
+	// run supercronic in other process
+	pwd, err := os.Getwd()
+	if err != nil {
+		logrus.Fatalf("Failed to get current working directory: %s", err.Error())
+		return
+	}
+
+	pattrs := &syscall.ProcAttr{
+		Dir: pwd,
+		Env: os.Environ(),
+		Files: []uintptr{
+			uintptr(syscall.Stdin),
+			uintptr(syscall.Stdout),
+			uintptr(syscall.Stderr),
+		},
+	}
+
+	pid, err := syscall.ForkExec(os.Args[0], os.Args, pattrs)
```

Comment:
With reaping being opt-out, we can explicitly tell the child process not to. This avoids extraneous warning messages about not being PID 1 included in the last suggestion.

```suggestion
	args := append(os.Args, "-no-reap")
	pid, err := syscall.ForkExec(args[0], args, pattrs)
```